netanalysis/ip/ip_info_test.py

```diff
@@ -4,54 +4,71 @@
 
 from . import ip_info as ii
 
+localhost_ip4 = ip("127.0.0.1")
+localhost_ip6 = ip("::1")
+
+# Google DNS addresses will be stably assigned to Google's AS"
+google_dns_ip4_8888 = ip("8.8.8.8")
```

Comment:
Make the constants ALL CAPS
https://google.github.io/styleguide/pyguide.html#3164-guidelines-derived-from-guidos-recommendations